### DISCUSSION TITLE
@remix-run/react has no exported member name useCatch

### DISCUSSION THREAD

**Discussion by BKG01** [remix-run/remix#9441] [Q&A]
W're getting this error when trying to use useCatch as per the docs https://remix.run/docs/en/1.19.3/guides/not-found#how-to-send-a-404..

Also I'm not clear how a Catchboundary is different to an ErrorBoundary? Thanks.

**Comment by kiliman** [comment]
They removed `useCatch` and `CatchBoundary` in v2.

I see your link is for 1.19.3. What about your Remix dependencies? If they're v2+ then you'll need to follow the upgrade guide.

https://remix.run/docs/en/main/start/v2#catchboundary-and-errorboundary

**Reply by BKG01** [reply]
Thanks that# clarifies things.  Thanks for your help
